app/views/sections/show.html.slim

```diff
@@ -24,28 +31,44 @@
           - @latest_videos.each do |video|
             .medium-4.cell
               = link_to(video)
-                = image_tag href="http://lorempixel.com/370/175/animals"
-              - unless video.nil?  
+                = image_tag video.image_url
+              - if video
                 h5 = video.name
                 p = video.description
   .subsections-container
     .grid-container
-      - @section.subsections.each do |subsection|
-        .subsection-block
+      - @subsections.each do |subsection|
+        .subsection-block id="subsection-#{subsection.id}"
           .title
             h3 = subsection.name
           - subsection.categories.each do |category|
             .category-block
               .category-title
-                h5 = category.name
+                h4 = category.name
                 p = category.description
-              / .category-latest
-              /   .grid-x.grid-padding-x
-              /     .medium-6.cell
-              /       = link_to(category.videos.last)
-              /         = image_tag src="http://lorempixel.com/400/300/animals"
-              /     .medium-6.cell
-              /       div
-              /         / h5 = category.videos.last.name
-              /         / p = category.videos.last.description
-              / .category-carousel
+              - unless category.videos.empty?
+                - category_latest_video = category.videos.last
```

Comment:
Always use `order` when you use `last` or `first`.

You can also implement `Category#last_video` to make it simpler for the views, like:

```
# models/category.rb

def last_video
  @last_video ||= videos.order(:created_at).last
end
```

(Remeber to add specs if you do that)